run_tests.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-pytest tests/*
+python -m pytest tests/*
```

Comment:
Lets pytest find imports properly (at least, I think so. It doesn't work without python -m)

Reply:
python -m pytest initiates the pytest module - I think you can also just call pytest and it'll do the same thing, although that could depend on your environment.

Reply:
something about adding python -m lets it see depdencies properly from the tests folder, it must be an environment thing.